gitlab/v4/objects/deployments.py

```diff
@@ -1,21 +1,23 @@
 from gitlab.base import RequiredOptional, RESTManager, RESTObject
 from gitlab.mixins import CreateMixin, RetrieveMixin, SaveMixin, UpdateMixin
 
+from .merge_requests import ProjectDeploymentMergeRequestManager  # noqa: F401
+
 __all__ = [
     "ProjectDeployment",
     "ProjectDeploymentManager",
 ]
 
 
 class ProjectDeployment(SaveMixin, RESTObject):
-    pass
+    _managers = (("mergerequests", "ProjectDeploymentMergeRequestManager"),)
 
 
 class ProjectDeploymentManager(RetrieveMixin, CreateMixin, UpdateMixin, RESTManager):
     _path = "/projects/%(project_id)s/deployments"
     _obj_cls = ProjectDeployment
     _from_parent_attrs = {"project_id": "id"}
-    _list_filters = ("order_by", "sort")
+    _list_filters = ("order_by", "sort", "environment", "status")
```

Comment:
What about `updated_after` and `updated_before` ?

https://docs.gitlab.com/ee/api/deployments.html#list-project-deployments

Reply:
Actually not sure why this was part of the original PR, but I've added the latest params.